buzzmobile/launch/lidar_test.launch

```diff
@@ -1,8 +1,6 @@
 <launch>
-    <group ns="buzzmobile">
-        <rosparam command="load" file="$(find buzzmobile)/constants.yaml"/>
-        <param name="hokuyo_node/port" value="/dev/ttyACM0"/>
-        <node pkg="buzzmobile" name="lidar_model" type="lidar_to_frame.py"/>
-        <node pkg="buzzmobile" name="frame_merger" type="frame_merger.py"/>
-    </group>
+    <rosparam command="load" file="$(find buzzmobile)/constants.yaml"/>
+	<include file="$(find buzzmobile)/launch/lidar_test/mock_sensors.launch"/>
+	<include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
+	<include file="$(find buzzmobile)/launch/lidar_test/visualization.launch"/>
```

Comment:
Actually can we have the dir structure be:

`buzzmobile/launch/main.launch`

`buzzmobile/launch/includes/` = main includes

`buzzmobile/launch/tmp/` = where test dirs go

`buzzmobile/launch/tmp/lidar_test` = main for `lidar_test`

etc?